runtime/gc_glue_java/ScavengerRootScanner.cpp

```diff
@@ -66,6 +66,8 @@ MM_ScavengerRootScanner::startContinuationProcessing(MM_EnvironmentBase *env)
 					if (!list->wasEmpty()) {
 						_scavengerDelegate->setShouldScavengeContinuationObjects(true);
 					}
+				} else {
+					list->backupList();
```

Comment:
While touching the file, fix a few coding standard problems like
- space between if/while and '('
- UDATA -> uintptr_t
- `*` next to variable, rather next to the type